tests/Authentication/MagicLinkTest.php

```diff
@@ -37,29 +37,29 @@ protected function setUp(): void
 
     public function testCanSeeMagicLinkForm(): void
     {
-        $result = $this->get(route_to('magic-link'));
+        $result = $this->get(url_to('magic-link'));
```

Comment:
Revert all the changes in this file. `$this->get()`, `$this->post()`, `$result->assertRedirectTo()` does not take absolute URL.

See
- https://codeigniter4.github.io/CodeIgniter4/testing/feature.html#requesting-a-page
- https://codeigniter4.github.io/CodeIgniter4/testing/response.html#assertredirectto